reference-artifacts/SAMPLE_CONFIGS/config.example.json

```diff
@@ -1252,6 +1259,50 @@
           "rsyslog-enforce-imdsv2": true,
           "rsyslog-root-volume-size": 100,
           "rsyslog-max-instance-age": 7
+        },
+        "opensearch-siem": {
+          "deploy": true,
+          "vpc-name": "Central",
+          "region": "${HOME_REGION}",          
+          "security-groups": [
+            {
+              "name": "opensearch-siem",
+              "inbound-rules": [
+                {
+                  "description": "Allow Traffic Inbound",
+                  "tcp-ports": [443],                  
+                  "source": "${RANGE-RESTRICT}"
+                }
+              ],
+              "outbound-rules": [
+                {
+                  "description": "All Outbound",
+                  "type": ["ALL"],
+                  "source": ["0.0.0.0/0"]
+                }
+              ]
+            }
+          ],
+          "app-subnets": [
+            {
+              "name": "App",
+              "az": "a"
+            },
+            {
+              "name": "App",
+              "az": "b"
+            }
+          ],          
+          "lambda-log-processing-role": "ASEA-OpenSearch-ProcessingRole",
+          "cognito-domain-prefix": "asea-siem-1234",
+          "opensearch-instance-type-main-nodes": "c6g.xlarge.search",
+          "opensearch-instance-type-data-nodes": "r6g.xlarge.search",
+          "opensearch-instance-role": "ASEA-OpenSearch-Role",
+          "opensearch-capacity-main-nodes": 3,
+          "opensearch-capacity-data-nodes": 4,
+          "opensearch-volume-size": 100,
+          "opensearch-configuration": "siem/opensearch-config.json",          
+          "event-processor-lambda-package": "siem/os-loader.zip"
```

Comment:
Question - is this the smallest usable OS cluster sizes (4*r6g.xl and 3*c5g.xl), any chance we can go smaller for frugal/new customers?  or you recommend we stick with this config?

Reply:
we can go smaller, but different instance types hasn't been tested. when selecting a smaller instance size, one would need to watch the health of the cluster. It also depends on how much activity is happening which impacts the indexing.